docs/beginner/tutorial3-pipeline/README.md

```diff
@@ -136,14 +139,14 @@ let render_pipeline = device.create_render_pipeline(&wgpu::RenderPipelineDescrip
         module: &fs_module,
         entry_point: "main",
     }),
+    // continued ...
```

Comment:
On my first reading, I was confused and thought the code block was over at this point as I typed along. I think it's more idiomatic and reads better to have the "continued" be at the end of a block instead of the beginning, but I can revert this if you disagree.